internal/routes/all_routes.go

```diff
@@ -57,6 +57,9 @@ func apiRouter() http.Handler {
 		r.Route("/sources", func(r chi.Router) {
 			r.Get("/", s.ListSources)
 			r.Route("/{ID}", func(r chi.Router) {
+				r.Get("/status", s.SourcesStatus)
+
+				// TODO move this to outside of /sources (see below)
```

Comment:
Why do you want to remove it outside?

Reply:
The plan for instance types is to preload them, see below, this is already implemented for Azure (`/instance_types/azure`) and I hope @avitova can work on refactoring AWS in the same way. According to testing, loading up instances from embedded YAML is < 1ms compared to 6 seconds of live loading.